test/parallel/test-http-get-pipeline-problem.js

```diff
@@ -38,13 +39,15 @@ const image = fixtures.readSync('/person.jpg');
 console.log(`image.length = ${image.length}`);
 
 const total = 10;
-let requests = 0;
-let responses = 0;
+const requestCountdown = new Countdown(total, common.mustCall(() =>
+  server.close()
```

Comment:
I could be wrong but I don't think we strictly need two separate countdowns in this test? We could just use the `responseCountdown` and have the `server.close()` be inside it.

Reply:
@apapirovski It works fine. Done.